tool/jt.rb

```diff
@@ -2373,6 +2374,8 @@ def bootstrap_toolchain
       remove_shared_compile_artifacts
     end
 
+    FileUtils.touch(Dir.glob('src/**/BuildInformation.java').first) if options.delete('--new-hash')
```

Comment:
I think it's nicer to specify the full path here rather than doing a glob over many directories/files, i.e. `src/shared/java/org/truffleruby/shared/BuildInformation.java`